Packs/PaloAltoNetworks_Threat_Vault/IncidentFields/incidentfield-ThreatVault_application.json

```
@@ -0,0 +1,77 @@
+{
+    "id": "incident_threatvaultbypaloaltonetworksapplicationsnew",
```

```suggestion
    "id": "incident_threatvaultbypaloaltonetworksapplications",
```
here and in other places